tdrs-frontend/src/components/FileUpload/FileUpload.jsx

```diff
@@ -194,7 +193,7 @@ function FileUpload({ section, setLocalAlertState }) {
         data-errormessage={INVALID_FILE_ERROR}
       />
       <div style={{ marginTop: '25px' }}>
-        {hasFile ? (
+        {hasFile && selectedFile?.id ? (
```

Comment:
If the ID is defined then we know it has been submitted to the backend and can be downloaded - thus we will only show the download button when this property is present.